.github/actions/docker-prepare/action.yml

```diff
@@ -0,0 +1,14 @@
+---
+name: 'docker-prepare'
+description: 'Install docker requirements'
+
+runs:
+  using: 'composite'
+  steps:
+  - uses: docker/setup-buildx-action@v2
+    with:
+      version: v0.7.1
+  - shell: bash
+    run: |
+      docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+      docker run --privileged --rm tonistiigi/binfmt --install arm64
```

Comment:
I am not sure this is needed at all

Reply:
I'm not sure either, I'll try out the build without these and let you know

Reply:
You're right, the arm64 builder is already available! (comparison of a pipeline [containing these lines](https://github.com/mvitale1989/elrond/actions/runs/4046566828) vs [with these lines removed](https://github.com/mvitale1989/elrond/actions/runs/4046653544))
I removed these lines